Iss 262 workspaces (#267)
MerginMaps/docs: src/manage/project-advanced.md

# How to Share, Transfer or Delete Projects
[[toc]]

## Share projects and manage user access

> You can share your <MainPlatformName /> project with others by inviting them to be [a member or a guest](./permissions/#workspace-members-and-guests) in your workspace. You can also make your <MainPlatformName /> project accessible to everyone by making it [public](#make-your-project-public-private).

Author: I think it might be a bit abstract to mention a "Share button" and "share dialogue" as the intention of this how to page is to basically guide them to the share button (even if they don't know the difference between a member and a guest yet)

To add users to a workspace:
1. Log into <AppDomainNameLink />
2. Select **Members** from the left panel and click on **Add People**
   ![add](../tutorials/working-collaboratively/web-workspace-add.png)
3. In the invite form, enter the email addresses of people you want to invite. If they already have a <MainPlatformName /> account, you can enter their username.
   Select if you invite them as **members** or as **guests**:
   - for **members**, select their [member role](./permissions/#workspace-and-project-permissions). Members have access to all projects in the workspace.
   - for **guests**, specify the projects to which they should have access. Guests can have [*read* or *write* permissions](./permissions/#workspace-and-project-permissions) to these projects.
  ![invite](./workspace-members-guests.png)

4. Click on **Invite** to send an invitation. An email with a link will be sent to the recipients. After they accept the invitation, they will become members or guests of your workspace.

> To manage workspace permissions, you have to be the **admin** or **owner** of the workspace. Navigate to **Members** in the left panel in <AppDomainNameLink /> to see the list of members and guests. Here you can change their roles and permissions or remove them from the workspace.

Reviewer: Aren't we already on the "members" page? 
```suggestion
To manage workspace permissions, you have to be the **admin** or **owner** of the workspace.  
On **Members** page you can see the list of members and guests. Here you can change their roles and permissions or remove them from the workspace.
```
